guides/cookbook.md

```diff
@@ -391,6 +391,217 @@ The result of `getBytes()` from a text or binary writer can subsequently
 be passed as the parameter to `makeReader()` in order to read the Ion data.
 </div>
 
+<div class="tabpane Python" markdown="1">
+Ion-Python provides two distict APIs for reading and writing Ion: a non-blocking event-based API, and
+a blocking `dump/load` API called `simpleion`, which is reminiscent of the popular `simplejson`
+JSON processing API. Where applicable, this cookbook will illustrate how to implement the examples using
+both APIs.
+
+The following examples show how Ion data can be read and written.
+
+#### Using simpleion
+
+```python
+from amazon.ion import simpleion
```

Comment:
The only real difference is whether you refer to the simpleion module as `simpleion` or `ion`. I'm inclined to leave it as-is in the cookbook since I've referred to this set of APIs as "simpleion" in the descriptions, not "ion" (which I expect could cause confusion).